File: python_backend/game_logic/ranges/get_range_func.py
from game_logic.ranges.special_defs import _special_defs


def get_range_func(piece_type):
    """return the function that gets the range for piece_type """
    return _special_defs[piece_type]


if __name__ == "__main__":
    print(get_range_func('Rook'))
    print(get_range_func('King'))
    print(get_range_func('Pawn'))

File: node.js/ranges/getRangeFunc.js
var  specialDefs  = require("./specialDefs");


function getRangeFunc(pieceType) {
    /*return the function that gets the range for gameData **/
    return specialDefs[pieceType]
}

module.exports = getRangeFunc;

if (require.main === module) {
    console.log(getRangeFunc('Rook'))
    console.log(getRangeFunc('King'))
    console.log(getRangeFunc('Pawn'))
}